### PR TITLE
Mobile User Profile Page

### PR DESCRIPTION
**Pull request by pxwxnvermx**
[Ticket](https://dimagi.atlassian.net/browse/CCCT-224)

- The Profile page is linked to the user's name on the Learn, Deliver, User, and Payment status tables on the opportunity details page.

image

### CODE REVIEW

File: commcare_connect/opportunity/views.py [hunk 1]
```diff
@@ -481,6 +483,34 @@ def payment_delete(request, org_slug=None, opp_id=None, access_id=None, pk=None)
...
+    user_visit_data = []
+    for user_visit in user_visits:
+        if not user_visit.location:
+            continue
+        lat, lng, elevation, precision = user_visit.location.split(" ")
+        user_visit_data.append(
+            dict(entity_name=user_visit.entity_name, visit_date=user_visit.visit_date.date(), lat=lat, lng=lng)
+        )
+    # user for centering the User visits map
+    lat_avg = reduce(lambda x, y: x + float(y["lat"]), user_visit_data, 0.0) / len(user_visit_data)
```
Comment (calellowitz) [inline commcare_connect/opportunity/views.py:499]: This will cause issues if `user_visit_data` is empty (no visists or none with locations)

**Review by calellowitz — CHANGES_REQUESTED** [review]
This looks fine besides fixing the one error

File: commcare_connect/opportunity/views.py (hunk 1 above)
Comment (pxwxnvermx) [reply commcare_connect/opportunity/views.py]: Fixed. https://github.com/dimagi/commcare-connect/pull/248/commits/17fbe13cd0b2b74c3388eac42cbf5283ad04273f

**Review by calellowitz — APPROVED** [review]
Approved and merged, but it might be nice to hide the map if there is no data

**Pull request merged**